cmd/system-shutdown/system-shutdown-utils.c

```diff
@@ -66,6 +66,27 @@ void die(const char *msg)
 	exit(1);
 }
 
+int read_reboot_arg(char *arg, int max_size)
+{
+	FILE *f;
+
+	*arg = '\0';
+
+	f = fopen("/run/systemd/reboot-param", "r");
```

Comment:
Can we get this properly documented with a link to the systemd source (a tagged version)? Also needs to be added in the go portion of this PR.